src/game/GUI.java

```diff
@@ -238,14 +238,18 @@ public boolean loadFile(File f) {
 			return false;
 		}
 		ObjectInputStream in = null;
+		Game newGame = null;
+		boolean error = false;
```

Comment:
No codepath actually depends on this field, and it isn't used for anything else (use forward code slicing to determine this)